lib/src/components/NxList/NxListEmpty.tsx

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+const NxListEmpty = () => {
+  return (
+    <li className="nx-list__item nx-list__item--empty">
+      <span className="nx-list__text">This list is empty.</span>
```

Comment:
Whether this stays as a component or gets built into `NxList`, we'll need this text to be customizable.

Reply:
> Whether this stays as a component or gets built into `NxList`, we'll need this text to be customizable.

This component has been removed. A prop `emptyMessage` on NxList customizes the empty text.
9c20728ac11326ee3289071ea1c0b76be6c3a745